extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -277,7 +279,7 @@ export class AmpConsent extends AMP.BaseElement {
       // Display the current instance
       this.currentDisplayInstance_ = instanceId;
       const uiElement = this.consentUI_[this.currentDisplayInstance_];
-      setImportantStyles(uiElement, {display: 'block'});
```

Comment:
I'm assuming this was just to undo the `display: none`, correct? You don't mean to actually force `display: block`?